ert_shared/ensemble_evaluator/entity/identifiers.py

```diff
@@ -109,3 +111,54 @@
     EVTYPE_ENSEMBLE_CANCELLED,
     EVTYPE_ENSEMBLE_FAILED,
 }
+
+PBS_CLUSTER_CLASS = "dask_jobqueue.LSFCluster"
+PBS_CLUSTER_KWARGS = {
+    "n_workers": 10,
+    "queue": "normal",
+    "project": "ERT-TEST",
+    "local_directory": "$TMPDIR",
+    "cores": 4,
+    "memory": "16GB",
+    "resource_spec": "select=1:ncpus=4:mem=16GB",
+}
+
+LSF_CLUSTER_CLASS = "dask_jobqueue.PBSCluster"
+LSF_CLUSTER_KWARGS = {
+    "queue": "mr",
+    "project": None,
+    "cores": 1,
+    "memory": "1GB",
+    "use_stdin": True,
+    "n_workers": 2,
+    "silence_logs": "debug",
+    "scheduler_options": {"port": find_open_port()},  # not nice, but file not __init__
+}
+
+LOCAL_EXECUTOR = "local"
+LSF_EXECUTOR = "lsf"
+PBS_EXECUTOR = "pbs"
+DASK_EXECUTORS = [
+    LOCAL_EXECUTOR,
+    LSF_EXECUTOR,
+    PBS_EXECUTOR,
+]
+
+LOCAL_DASK_EXECUTOR_KWARGS = {
+    "silence_logs": "debug",
+    "scheduler_options": {"port": find_open_port()},
+}
+
+PBS_DASK_EXECUTOR_KWARGS = {
+    "cluster_class": PBS_CLUSTER_CLASS,
+    "cluster_kwargs": PBS_CLUSTER_KWARGS,
+    "debug": True,
+}
+
+LSF_DASK_EXECUTOR_KWARGS = {
+    "cluster_class": LSF_CLUSTER_CLASS,
+    "cluster_kwargs": LSF_CLUSTER_KWARGS,
+    "debug": True,
+}
```

Comment:
These do not belong in the identifiers, which probably should be renamed to `constants.py`?

Anyway, the default prefect configurations could go in a `ensemble_evaluator/prefect` folder.

Reply:
I created a new module ensemble_evaluator/prefect/defaults.py where these default configurations live